README.md

```diff
@@ -1,18 +1,60 @@
-<a href="https://github.com/matchai/spacefish">
-  <img alt="spacefish fish shell theme" src="./images/banner.svg" width="100%">
-</a>
+<h1 align="center">
+  <a href="https://github.com/matchai/spacefish">
+    <img alt="spacefish fish shell theme" src="./images/banner.svg" width="100%">
+  </a>
+  <br>
+  <br> 🚀🐟 Spacefish <br>
```

Comment:
`<br> Spacefish 🚀🐟 <br>`

Looks a bit nicer since you don't have the text weirdly placed.